index.html

```diff
@@ -31,36 +31,36 @@
 
             <ul class="nav top-nav d-flex justify-content-end">
 
-                <!-- NEWS -->
-                <li class="nav-item">
-                    <a class="nav-link" href="pages/news.html">NEWS</a>
-                </li>
-                <!-- DOCS -->
-                <li class="nav-item">
-                    <a class="nav-link" href="https://www.uyuni-project.org/uyuni-docs/">DOCS</a>
-                </li>
                 <!-- Dropdown for available Uyuni versions -->
                 <li class="nav-item dropdown">
                     <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-                        VERSION
+                        DOWNLOAD
```

Comment:
This menu is not about any download. It leads to information about Stable or Master, including release notes, doc, setup, etc. If we want to avoid `VERSION`, I'd rather call this `SETUP`.

Reply:
NOK. People are looking for a "download" button, not for "version", "setup" or anything else.

Reply:
Agreed, but then let's not change the linked pages to add the installation instructions first.

Release notes and doc first, even if it's only to increase 0,00001% the changes that people will have a look first.